### ISSUE TITLE
split two metrics

### ISSUE THREAD

**Issue by gjeee** [rapmue/iota-munin-plugin#1]
hi, 

can u split the graph below in 2 separate graphs? it will give better contrast. tx!

![image](https://user-images.githubusercontent.com/13155183/53275293-bfda7600-36fa-11e9-8500-5b40f3f366a4.png)

now the second graph doesn't mean much :-(

**Comment by rapmue** [comment]
Hi,

If I understand your question correct, the scale of the two lines are too big apart?

I'll try to make different scales.

**Comment by gjeee** [comment]
yeah i mean txToRequest always drifts back to 0 (if it works correctly) and tips goes from 0 to 10k (maximum capacity)

so hard to get contrast between the two graphs by putting them on the same scale.

**Comment by rapmue** [comment]
Ok, I changed the linear scale to a logarithmic one. The difference should be more visible now. I'm just playing around with this setting.
